### PR TITLE
Ft top authors

### PR DESCRIPTION
**Pull request by Elochukwu3**
Made some changes and add the top authors page

### CODE REVIEW

File: src/components/ProgressBar.jsx [hunk 1]
```diff
@@ -24,14 +31,12 @@ const dispatch = useDispatch();
...
+  <Layout>
     <section className="flex w-full justify-center h-full">
-      {/* <div className="w-1/2  h-full max-md:w-full px-5 py-4 "> */}
-      <div className="w-1/2  h-full max-md:w-full py-4 ">
+      <div className="w-3/6  h-full max-md:w-full py-4 ">
       <div className="flex gap-10 items-center">
         <div onClick={handlePrev}>
-          <ArrowLeft size={24} />
-    
+          <ArrowLeft size={24} className='cursor-pointer'/>
```
Comment (simdi) [inline src/components/ProgressBar.jsx]: Please don't use static values. Eg. size={24}

File: src/components/home-screen/DiscoverHeader.jsx [hunk 2]
```diff
@@ -10,7 +11,7 @@ const DiscoverHeader = ({caption, page}) => {
         <Link to={page} className="text-blue-600 block text-xl text-clip">View all</ Link>{" "}
         <span>
          
-          <ArrowRight color="blue" size={24}/>
```
Comment (simdi) [inline src/components/home-screen/DiscoverHeader.jsx:13]: Same here. Don't use static values.

File: src/components/home-screen/SearchInput.jsx [hunk 3]
```diff
@@ -1,12 +1,13 @@
 import { MagnifyingGlass  } from "phosphor-react";
+import { cssValues } from "../../assets/staticValues";
 
 
 
 const SearchInput = ({setSearchValue}) => {
    
   return (
-    <form className='flex items-center bg-red-500 w-full max-md:w-full  bg-[#f5f5f5] px-2 rounded-md overflow-hidden'>
```
Comment (simdi) [inline src/components/home-screen/SearchInput.jsx:8]: Defined this color `#f5f5f5` in the tailwind config.

File: src/components/home-screen/SearchInput.jsx [hunk 4]
```diff
@@ -1,12 +1,13 @@
...
-    <MagnifyingGlass size={24} weight="light" />
```
Comment (simdi) [inline src/components/home-screen/SearchInput.jsx:9]: Same here. Remove static values.

File: src/components/home-screen/SearchPeople.jsx [hunk 5]
```diff
@@ -1,27 +1,53 @@
...
+          {
+             verify && <img src="https://img.icons8.com/fluency/48/null/verified-badge.png" className="w-5 h-5"/>
+          }
+          
         </div>
+        <p className="text-gray-400">{email}</p>
+      </div>
+       </div>
         <button
-          className="bg-[#6949ff] w-content px-3 py-1 rounded-[20px] text-white"
```
Comment (simdi) [inline src/components/home-screen/SearchPeople.jsx:18]: Define this color `#6949ff` in the tailwind config.

Comment (simdi) [reply src/components/home-screen/SearchPeople.jsx:18]: You can also define this `20px` in the tailwind config.

File: src/components/home-screen/ShareContacts.jsx [hunk 6]
```diff
@@ -2,16 +2,16 @@ import { CaretRight} from "phosphor-react";
...
     return (
-      <div className='w-full flex items-center capitalize justify-between border-b-2 border-[#f5f5f5] py-4 max-sm:gap-4'>
+      <div className='w-full flex items-center capitalize justify-between  max-sm:gap-4'>
           <div className='pl-3 max-sm:pl-0'>
-              <img src={image} alt="" />
+              <img src={image} alt="" className="w-11 h-11"/>
           </div>
-          <div className='space-y-2'>
-              <h2 className='text-xl  font-[700]'>{caption}</h2>
-              <p className='truncate w-[200px] max-sm:w-w-[190px] font-serif text-[18px] text-[grey]'>{detail}</p>
```
Comment (simdi) [inline src/components/home-screen/ShareContacts.jsx:11]: Same here.

File: src/components/home-screen/ShareContacts.jsx [hunk 7]
```diff
@@ -2,16 +2,16 @@ import { CaretRight} from "phosphor-react";
...
+          <div className='space-y-2 text-left'>
+              <h2 className='text-xl  font-bold'>{caption}</h2>
+              <p className=' font-serif text-lg text-[grey] truncate max-sm:w-48'>{detail}</p>
           </div>
   
-          <CaretRight size={24}/>
```
Comment (simdi) [inline src/components/home-screen/ShareContacts.jsx:14]: Same here.

File: src/components/home-screen/TopColImages.jsx [hunk 8]
```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+
+const TopColImages = ({ image, caption }) => {
+    return (
+      <div className="h-40 space-y-4 rounded-3xl overflow-hidden w-30 shadow-xl mx-3">
+        <div className="h-full w-full overflow-hidden  relative">
+          <img src={image} alt="" className="h-full w-full" />
+          <div className="absolute text-white px-5 font-medium text-2xl w-full bg-[#00000022] py-3 border-unset bottom-0 hover:bg-[#00000099]">
```
Comment (simdi) [inline src/components/home-screen/TopColImages.jsx]: Same here.

File: src/components/home-screen/TopCollection.jsx [hunk 9]
```diff
@@ -33,30 +34,22 @@ const TopCollection = () => {
...
     </div>
   );
 };
 
-const TopCollectionsImages = ({ image }) => {
-  return (
-    <div className="space-y-4 rounded-3xl overflow-hidden w-30 shadow-xl mx-3">
-      <div className="h-40 w-full overflow-hidden  relative">
-        <img src={image} alt="" className="h-full w-full" />
-        <div className="absolute text-white px-5 font-medium text-2xl w-full bg-[#00000022] py-3 border-unset bottom-0 hover:bg-[#00000099]">
```
Comment (simdi) [inline src/components/home-screen/TopCollection.jsx:54]: Same here.

File: src/components/intro/GetStartedButton.jsx [hunk 10]
```diff
@@ -2,19 +2,15 @@ import React from "react";
...
-  /**
-   * !in the i already have account button put link to your created account page
-   *
-   */
   return (
-    <div className="uppercase text-white space-y-4 mt-16 w-full flex items-center flex-col justify-center">
+    <div className="uppercase text-white space-y-4 mt-10 max-sm:mt-15 max-sm:space-y-5 w-full flex items-center flex-col justify-center">
       <Link to={"/occupation"} className="w-[270px] max-sm:w-[90%]">
-        <button className=" shadow-sm shadow-[#6949ff] px-3 w-full uppercase text-sm bg-[#6949ff] block py-3 rounded-[30px]  hover:opacity-75 hover:text-[13px]">
+        <button className=" shadow-sm shadow-[#6949ff] px-3 w-full uppercase text-sm bg-[#6949ff] block py-3 rounded-[30px]  hover:opacity-75 hover:bg-white hover:text-[#6949ff] hover:border-2 border-[#6949ff]">
```
Comment (simdi) [inline src/components/intro/GetStartedButton.jsx:8]: Same here.

File: src/components/intro/GetStartedButton.jsx [hunk 11]
```diff
@@ -2,19 +2,15 @@ import React from "react";
...
           get started
         </button>
       </Link>
       <Link to={"/occupation"} className="w-[270px] max-sm:w-[90%]">
-        <button className=" shadow-sm shadow-[#bbb] px-3 w-full uppercase text-sm text-[#6949ff]  py-3 rounded-[30px] block  bg-[#f0edff] hover:opacity-75 hover:text-[13px]">
+        <button className=" shadow-sm shadow-[#bbb] px-3 w-full uppercase text-sm text-[#6949ff]  py-3 rounded-[30px] block  bg-[#f0edff] hover:opacity-75 border-2 border-[#6949ff]">
```
Comment (simdi) [inline src/components/intro/GetStartedButton.jsx:13]: Same here.

File: src/pages/FindFriends.jsx [hunk 12]
```diff
@@ -29,22 +29,29 @@ const FindFriends = () => {
           <div className="text-2xl capitalize font-[600]">find friends</div>
         </div>
         <SearchInput setSearchValue={setSearchValue} />
-        <article className="shadow-md p-4   rounded-md space-y-5">
+        <article className="p-4  rounded-[25px] space-y-5 shadow-lg">
```
Comment (simdi) [inline src/pages/FindFriends.jsx]: Same here. Define this value `25px` in the tailwind config.

**Review by simdi — CHANGES_REQUESTED** [review]

**Review by Elochukwu3 — COMMENTED** [review]
Modified the previous static values and made use of tailwind config file for adjustment and new values

File: src/pages/FindFriends.jsx (hunk 12 above)
Comment (Elochukwu3) [reply src/pages/FindFriends.jsx]: done sir

File: tailwind.config.js [hunk 13]
```diff
@@ -4,7 +4,20 @@ module.exports = {
     "./src/**/*.{js,jsx,ts,tsx}",
   ],
   theme: {
-    extend: {},
+    extend: {
+      transitionDuration: {
+        '900': '900ms',
+      },
+      colors:{
+        sbg: "#6949ff",
```
Comment (simdi) [inline tailwind.config.js]: Why don't you use the actual color names?
It would be descriptive for anyone seeing this in the future.

**Review by simdi — CHANGES_REQUESTED** [review]

**Pull request merged**